README.md

```diff
@@ -1,147 +1,7 @@
-| [Texas Instruments AM57xx](http://www.ti.com/processors/sitara/arm-cortex-a15/am57x/overview.html)|`PLATFORM=ti-am57xx`| Yes | ![Actively maintained](documentation/images/green.svg) |
-| [Texas Instruments AM43xx](http://www.ti.com/processors/sitara/arm-cortex-a9/am438x/overview.html)|`PLATFORM=ti-am43xx`| Yes | ![Actively maintained](documentation/images/green.svg) |
-| [Xilinx Zynq 7000 ZC702](http://www.xilinx.com/products/boards-and-kits/ek-z7-zc702-g.html)|`PLATFORM=zynq7k-zc702`| Yes | ![Not maintained](documentation/images/red.svg) v2.3.0 |
-| [Xilinx Zynq UltraScale+ MPSOC](http://www.xilinx.com/products/silicon-devices/soc/zynq-ultrascale-mpsoc.html)|`PLATFORM=zynqmp-zcu102`| Yes | ![Not maintained](documentation/images/red.svg) v2.4.0 |
-| [Spreadtrum SC9860](http://www.spreadtrum.com/en/SC9860GV.html)|`PLATFORM=sprd-sc9860`| No | ![Not maintained](documentation/images/red.svg) v2.1.0 |
-
----
-## 4. Get and build OP-TEE software
-Please see [build] for instructions how to run OP-TEE on various devices.
-
----
-## 5. Coding standards
-In this project we are trying to adhere to the same coding convention as used in
-the Linux kernel (see [CodingStyle]). We achieve this by running [checkpatch]
-from Linux kernel. However there are a few exceptions that we had to make since
-the code also follows GlobalPlatform standards. The exceptions are as follows:
-
--	CamelCase for GlobalPlatform types are allowed.
--	And we also exclude checking third party code that we might use in this
-	project, such as LibTomCrypt, MPA, newlib (not in this particular git, but
-	those are also part of the complete TEE solution, see
-	[Notice.md](Notice.md#repository-structure). The reason for
-	excluding and not fixing third party code is because we would probably
-	deviate too much from upstream and therefore it would be hard to rebase
-	against those projects later on and we don't expect that it is easy to
-	convince other software projects to change coding style.
-	Automatic variables should always be initialized. Mixed declarations
-	and statements are allowed, and may be used to avoid assigning useless
-	values. Please leave one blank line before and after such declarations.
-
-Regarding the checkpatch tool, it is not included directly into this project.
-Please use checkpatch.pl from the Linux kernel git in combination with the
-local [checkpatch script].
-
-There are also targets for common use cases in the [Makefile](Makefile):
-
-```
-make checkpatch			#check staging and working area
-make checkpatch-staging #check staging area (added, but not committed files)
-make checkpatch-working #check working area (modified, but not added files)
-```
-
-[build]: https://github.com/OP-TEE/build
-[checkpatch script]: scripts/checkpatch.sh
-[checkpatch]: http://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/tree/scripts/checkpatch.pl
-[CodingStyle]: https://www.kernel.org/doc/Documentation/process/coding-style.rst
+// OP-TEE core maintainers, 2019-02-21
```

Comment:
Minor - this line does not seem super useful to me. The date can best be found in the commit itself.

Reply:
Yeah, you're probably right. I'll remove the date when adding the tag.